lib/amazonka/src/Amazonka/Presign.hs

```diff
@@ -11,6 +11,9 @@
 --
```

Comment:
Please update the copyright year in the haddock, and remove references to `'MonadIO'` (removed here) and to `AWS specific transformers` (which have been gone for a long time).